app/forms/FixedEstablishmentFormProviderVariantB.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.mappings.Mappings
+import play.api.data.Form
+
+import javax.inject.Singleton
+
+@Singleton
+class FixedEstablishmentFormProviderVariantB extends Mappings {
+
+  def apply(): Form[Boolean] =
+    Form(
+      "value" -> boolean("variantB.fixedEstablishment.error.required")
```

Comment:
There's an issue with using a boolean form on a radio button with more than 2 radios. If two buttons have the same value, when you select one and then go back to the page you get an ISE because the form doesn't know how to decode it. I think this will need a new form with separate values for each radio and we handle the conversion to a boolean when the answer is saved. We might have to store this answer twice: Once as a boolean which will be submitted as part of the registration and once as UK/IOM/Other to be used to populate the answers if the user goes back to the page. The second user answer wouldn't be submitted as part of the registration json. There is also a separate question about how this will be displayed on the final check your answer page